spec/models/manageiq/providers/autosde/inventory_spec.rb

```diff
@@ -23,5 +23,11 @@
     expect(ems.cloud_volumes).to_not be_empty
 
     expect(ems.cloud_volumes.first).to be_instance_of(ManageIQ::Providers::Autosde::StorageManager::CloudVolume)
+
+    # wwpn candidates
+    expect(ems.wwpn_candidates).to_not be_empty
```

Comment:
Better than `expect().not_to be_empty` would be to check `expect(ems.wwpn_candidates.count).to eq(whatever)`